src/components/Button.js

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ name }) => (
-  <button type="button">{name}</button>
+const Button = ({ name, handleClick }) => (
+  <button
+    type="button"
+    onClick={() => { handleClick(name); }}
+  >
+    {name}
+  </button>
 );
```

Comment:
- [ ] [Optional] it should implement `handleClick` function ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png))

```javascript
handleClick = (buttonName) => {
  this.props.clickHandler(buttonName);
};
```

**_this is only optional because it wasn't requested by the previous reviewer, but is originally a requirement, please take the time to add this simple function_**